docs/triggers.md

```diff
@@ -4,33 +4,26 @@ linkTitle: "Trigger"
 weight: 9
 ---
 -->
-# Triggers
+# `Triggers`
 
-A `Trigger` is resource that combines `TriggerTemplate`, `TriggerBindings` and `interceptors`. The `Trigger` is processed by EventListener which referenced it when it receives an incoming.
+A `Trigger` specifies what happens when the [`EventListener`](./eventlisteners.md) detects an event. A `Trigger` specifies a [`TriggerTemplate`](./triggertemplates.md),
+a [`TriggerBinding`](./triggerbindings.md), and optionally an [`Interceptor`](./eventlisteners.md#interceptors).
 
-- [Triggers](#triggers)
-  - [Syntax](#syntax)
-    - [template](#template)
+## Structure of a `Trigger`
 
-## Syntax
-
-To define a configuration file for an `Trigger` resource, you can specify
-the following fields:
+When creating a `Trigger` definition you must specify the required fields and can also specify any of the optional fields listed below:
 
 - Required:
-  - [`apiVersion`][kubernetes-overview] - Specifies the API version, for example
-    `triggers.tekton.dev/v1alpha1`.
-  - [`kind`][kubernetes-overview] - Specifies the `Trigger` resource
-    object.
-  - [`metadata`][kubernetes-overview] - Specifies data to uniquely identify the
-    `Trigger` resource object, for example a `name`.
-  - [`spec`][kubernetes-overview] - Specifies the configuration information for
-    your Trigger resource object. The spec include:
+  - [`apiVersion`][kubernetes-overview] - Specifies the API version; for example `triggers.tekton.dev/v1alpha1`.
+  - [`kind`][kubernetes-overview] - Specifies that this resource object is a `Trigger` object.
+  - [`metadata`][kubernetes-overview] - Specifies metadata to uniquely identify this `Trigger` object; for example a `name`.
+  - [`spec`][kubernetes-overview] - Specifies the configuration information for this Trigger object, including:
     - [`bindings`] - (Optional) A list of bindings to use. Can either be a reference to existing `TriggerBinding` resources or embedded name/value pairs.
```

Comment:
We should update the bindings description to match the others too: .Specfies a list of bindings to use etc.

Reply:
Fixed!